Automattic/jetpack
# Added copy post url button

Fixes https://github.com/Automattic/jetpack/issues/12927

#### Changes proposed in this Pull Request:
This PR adds a copy link button in the share this section, which copies the link of the post to the clipboard. 


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1. Go to Jetpack->Settings->Sharing and click on configure your sharing buttons. 
2. Make sure that you see **Copy Link** as one of the options in the available services list.
3. Move **Copy Link** to the list of enabled services. 
4. Click on "Save Changes" and visit any post and make sure that the "Copy Link" shows up in the share this section.
5. Click on the "Copy Link" button and observe that the toast, "coped post link to the clipboard" appears. 
6. Come back to the sharing buttons screen and change the button style to each of the options available (i.e Icon+Text, Icon Only, Text Only, and Official Buttons) and make sure the changes get reflected in the live preview section. 
7.  Repeat step 6 and go to any post and observe that the "Copy Link" button loads correctly corresponding to the selected button style. 

#### Video demo

https://user-images.githubusercontent.com/6594561/147484866-61c4dc3d-8fad-4dc1-83f0-8cb2199d8e0c.mov
